traffic_portal/test/integration/PageObjects/SideNavigationPage.po.ts

```diff
@@ -158,10 +158,9 @@ export class SideNavigationPage extends BasePage{
         await browser.actions().mouseMove(this.lnkServiceCategories).perform();
         await browser.actions().click(this.lnkServiceCategories).perform();
     }
-    async NavigateToCertExpirationsPage(){
-        await browser.wait(ExpectedConditions.visibilityOf(this.lnkCertExpirations), 2000);
-        await browser.actions().mouseMove(this.lnkCertExpirations).perform();
-        await browser.actions().click(this.lnkCertExpirations).perform();
+    // NavigateToCertExpirationsPage verifies that the link to the Certificate Expirations page is clickable.
```

Comment:
JSDoc comments need to be started with `/**` (and therefore ended with `*/`) in order to meet the spec. IDEs and the `jsdoc` program that they use won't pick it up otherwise.